src/styles/card.css

```diff
@@ -0,0 +1,120 @@
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 16px;
+  background: #ffff;
+  width: 200px;
+}
+
+.card__img {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin: 16px 3px 40px;
+}
+
+.card__product {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 4px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 23px;
+}
+
+.review__rating {
+  display: flex;
+}
+
+.stars {
```

Comment:
Create a separate stars.css file for it
![image](https://user-images.githubusercontent.com/96409563/205729871-ef99589d-c398-421c-ac81-9cfb67ba9e25.png)